src/foam/u2/view/RadioView.js

```diff
@@ -76,8 +79,7 @@ foam.CLASS({
 
       this.add(this.choices.map(function(c) {
         return this.E('div').
-          // TODO: why is the radio item getting assigned the same class as the radio whole
-          addClass(this.myClass()).
+          addClass('choice').
```

Comment:
Is this used? Can we just remove this line?

Reply:
If you are referring to `addClass('choice')`, then yes we do to style each choice

Reply:
agree - the extra spacing seems unnecessary - and ruins our standard

Reply:
@Nauna hmm, not sure what spacing you are referring to. Could you please elaborate?

Reply:
css for choice =
    ^ .foam-u2-view-RadioView .choice {
      margin-bottom: 16px;
      font-size: 16px;
    }
    
  Im referencing margin-bottom: 16px;
    
  But hmmm - perhaps that isn't the spacing in image...
  
![image](https://user-images.githubusercontent.com/10802216/107792953-c0270e00-6d23-11eb-9fc7-11178b984a30.png)

Reply:
Still think it looks good - and approve these changes

Reply:
Edit AllViews.js to include a RadioView and see how it looks in a regular DetailView.

Reply:
@Nauna that margin-bottom: 16px was added when displayed vertically I assume. I can take that out when the choices are being displayed horizontally, or from the last choice if displayed vertically.

Reply:
@kgrgreer 

radioView look after moving css to RadioView:

![Screen Shot 2021-02-12 at 11 47 15 AM](https://user-images.githubusercontent.com/58435071/107797029-7987e280-6d28-11eb-80c2-3291f25df461.png)